realsense_camera/launch/f200_nodelet_default.launch

```diff
@@ -4,7 +4,34 @@
   <arg name="camera_type"  default="F200" /> <!-- Type of camera -->
   <arg name="serial_no"    default="" />
   <arg name="usb_port_id"  default="" /> <!-- USB "Bus#-Port#" -->
-  <arg name="manager"      value="$(arg camera)_nodelet_manager" />
+  <arg name="manager"      value="nodelet_manager" />
```

Comment:
Yes, we have defaults only for those fields that are likely to be changed by users.